apps/protocol-frontend/src/components/ProfileForm.tsx

```diff
@@ -74,6 +80,34 @@ const ProfileForm = () => {
     });
   }, [userData, disconnectLinearUser]);
 
+  const disconnectDiscordOnClick = useCallback(async () => {
```

Comment:
We should wrap this logic in the hook? I don't think we need the useCallback here